src/Microsoft.ML.Analyzer/Microsoft.ML.Analyzer.csproj

```diff
@@ -3,6 +3,9 @@
   <PropertyGroup>
     <TargetFramework>netstandard1.3</TargetFramework>
     <IncludeAnalyzerInPackage>Microsoft.ML</IncludeAnalyzerInPackage>
+
+    <!-- No public surface in this assembly -->
+    <RunApiCompat>false</RunApiCompat>
```

Comment:
This shouldn't be necessary since ML.Analyzer isn't listed as a stable project. Can be reverted.